pages/linux/vigr.md

```diff
@@ -0,0 +1,12 @@
+# vigr
+
+> Edit the group file.
+> More information: <https://manned.org/vigr>.
+
+- Edit the group file:
+
+`vigr`
+
+- Display the current version of `vigr`:
```

Comment:
The examples are about `vigr` so
```suggestion
- Display version:
```

Reply:
Thanks for the feedback love this project.